convert_state.py

```diff
@@ -1,60 +1,65 @@
+"""This code converts written state names to the 2 letter abbreviations, or the reverse"""
```

Comment:
"""
This code converts written state names to the 2 letter abbreviations, or the reverse
"""